### PR TITLE
[zh] resync 1.27 policy resources/ & endpoint-slice-v1.md

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
This PR resync pages in floder `policy resources` and page `content/zh-cn/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1.md` referring to upstream en page.

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md [hunk 1]
```diff
@@ -39,9 +39,9 @@ NetworkPolicy 描述针对一组 Pod 所允许的网络流量。
 
 - **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
   <!--
-  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  Standard list metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
   -->
-  标准的对象元数据。更多信息：
+  标准的列表元数据。更多信息：
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md:44]: 這裡的英文是錯的

File: content/zh-cn/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1.md [hunk 2]
```diff
@@ -286,12 +283,25 @@ EndpointSlice 是实现某 Service 的端点的子集。一个 Service 可以有
...
-    此端口的应用层协议。此字段遵循标准的 Kubernetes Label 句法。
-    不带前缀的名称是 IANA 标准服务的保留名称（参见 RFC-6335 和 https://www.iana.org/assignments/service-names）。
-    非标准协议应该使用带前缀的名称，例如 mycompany.com/my-custom-protocol。
+    * Kubernetes-defined prefixed names:
+      * 'kubernetes.io/h2c' - HTTP/2 over cleartext as described in https://www.rfc-editor.org/rfc/rfc7540
+    
+    * Other protocols should use implementation-defined prefixed names such as mycompany.com/my-custom-protocol.
+    -->
+
+    此端口的应用层协议。这被用作提示，告诉实现者在他们理解的协议中提供更丰富的行为。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1.md:296]:
```suggestion
    此端口的应用层协议。字段值被用作提示，允许协议实现为其所理解的协议提供更丰富的行为。
```

File: content/zh-cn/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1.md [hunk 3]
```diff
@@ -286,12 +283,25 @@ EndpointSlice 是实现某 Service 的端点的子集。一个 Service 可以有
...
+    此字段遵循标准的 Kubernetes Label 句法。有效的取值是：
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1.md:297]:
```suggestion
    此字段遵循标准的 Kubernetes 标签句法。有效的取值是：
```

**Review by howieyuen — COMMENTED** [review]
/lgtm

**Pull request merged**